packages/route/src/Infrastructure/Doctrine/Repository/RouteRepository.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Route\Infrastructure\Doctrine\Repository;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\Query;
+use Doctrine\ORM\QueryBuilder;
+use Sulu\Route\Domain\Model\Route;
+use Sulu\Route\Domain\Repository\RouteRepositoryInterface;
+
+/**
+ * @phpstan-import-type RouteFilter from \Sulu\Route\Domain\Repository\RouteRepositoryInterface
```

Comment:
```suggestion
 * @phpstan-import-type RouteFilter from RouteRepositoryInterface
```

I think you can omit the full path